malos/heartbeat.proto

```diff
@@ -63,3 +64,7 @@ enum Sensors {
   UV = 8;
   ALTITUDE = 9;
 }
+
+enum Integrations {
+  GPIO = 0;
```

Comment:
Guess we can keep this one like that for consistency but other implementations can indeed take advantage of that change